e2e/playwright/share-link.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { expect, test } from '@e2e/playwright/zoo-test'
+
+const isWindows =
+  navigator.platform === 'Windows' || navigator.platform === 'Win32'
+test.describe('Share link tests', () => {
```

Comment:
Let's add an `@electron` tag to the whole group. You can likely remove your `process.env.PLATFORM !== 'web'` check as well.